packages/lodestar/src/chain/blocks/process.ts

```diff
@@ -29,6 +29,10 @@ export function processBlock(
   return (source) => {
     return (async function* () {
       for await (const job of source) {
+        if (job.signedBlock.message.slot > forkChoice.getTime()) {
```

Comment:
I think we should just pass in our `IBeaconClock` and use `clock.currentSlot` here.
I have a branch where I did the same.